packages/web-components/src/components/ic-menu/ic-menu.tsx

```diff
@@ -909,6 +914,7 @@ export class Menu {
     this.keyboardNav = false;
     this.menu.focus();
     this.emitSelectAll();
```

Comment:
I believe that `this.emitSelectAll` is used on [line 773](https://github.com/mi6/ic-ui-kit/blob/d8f4ce2e767aeeb18e03dc1cc14581e6e9ad3ae8/packages/web-components/src/components/ic-menu/ic-menu.tsx#L773) when a user pressed ctrl + a, so the icClear function should probably be added there as well.

In that case, might be worth combining the functions into one? Not necessary but may help for readability later down the line.